### ISSUE TITLE
2.0.0 main.js and vendors.js served as localhost index

### ISSUE THREAD

**Issue by HugoHeneault** [ionic-team/ionic-app-scripts#1101]
#### Short description of the problem:
From the 1.3.12 I updated to 2.0.0 but `main.js` and `vendors.js` files seems to return the same response as `http://localhost:8100`.

#### What behavior are you expecting?
Theses files should be served as the real ones, not some html :)

**Steps to reproduce:**
1. run `npm install @ionic/app-scripts@latest --save-dev` from a project with 1.3.12
2. add `<script src="build/vendor.js"></script>` before main.js

**Which @ionic/app-scripts version are you using?**
2.0.0

**Other information:** (e.g. stacktraces, related issues, suggestions how to fix, stackoverflow links, forum links, etc)
````
global packages:

    @ionic/cli-utils : 1.4.0
    Cordova CLI      : 7.0.1
    Ionic CLI        : 3.4.0

local packages:

    @ionic/app-scripts              : 2.0.0
    @ionic/cli-plugin-cordova       : 1.4.0
    @ionic/cli-plugin-ionic-angular : 1.3.1
    Cordova Platforms               : none
    Ionic Framework                 : ionic-angular 3.5.0

System:

    Node       : v8.1.3
    OS         : macOS Sierra
    Xcode      : not installed
    ios-deploy : not installed
    ios-sim    : not installed
    npm        : 5.1.0
````

**Comment by danbucholtz** [comment]
I'm not sure I'm following, can you explain this to me in more detail?

Thanks,
Dan

**Comment by janpio** [comment]
Everything "unkown" was served index.html in 2.0.0. As you know this included the proxy paths, most probably for @HugoHeneault somehow also the default files. I had something similar when I opened localhost:8100/foo/ - it tried to load /foo/main.js which then contained index.html content.

**Comment by danbucholtz** [comment]
Ahhh, I understand. This is fixed in `2.0.1`. There was a PR that we merged that was likely the culprit. I reverted it.

Thanks,
Dan

**Comment by HugoHeneault** [comment]
Yeah it's fixed! Thank you very much @danbucholtz
